TodoAppWithSpringboot/src/main/java/com/webonise/service/TodoService.java

```diff
@@ -0,0 +1,64 @@
+package com.webonise.service;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.webonise.dao.TodoDao;
+import com.webonise.model.Todo;
+import lombok.extern.slf4j.Slf4j;
+
+@Service
+@Slf4j
+public class TodoService {
+
+	@Autowired
+	private TodoDao todoDao;
+
+	public List<Todo> findAll() throws Exception {
+		try {
+			return (List<Todo>) todoDao.findAll();
+		} catch (NullPointerException ex) {
```

Comment:
This can be more proper just check `null` and `isEmpty` rather than throwing an exception. 
Also, an exception handling for NullPointerException should be more sophisticated and customized
exception should be thrown something `DataNotFound`.